gen3_tracker/meta/skeleton.py

```diff
@@ -55,6 +56,21 @@ def meta_index():
 
     return id_dict
 
+def get_data_from_meta() -> Generator[int, None, None]:
+    """Read all the ndjson files in the `META` directory and return a generator that produces all records"""
+    fhir_list = []
+    meta_dir = pathlib.Path('META')
+
+    for file in meta_dir.glob('*.ndjson'):
+        with open(file, 'r') as f:
+            for line in f:
+                record = orjson.loads(line)
+                resource_type = record.get('resourceType')
+                if resource_type == 'Bundle':
+                    break
```

Comment:
Why does this break here?

Reply:
@matthewpeterkort Looks good so far.   I had some relatively untested code to automatically create bundles based here https://github.com/ACED-IDP/gen3_util/blob/feature/fhir-bundle-submission/gen3_tracker/meta/skeleton.py#L299.  take a look, let me know what you think

Reply:
> Why does this break here?

Because at the moment the ETL pod does not support bundle submission, and that function ```get_data_from_meta(``` is only being used for gathering data files to be submitted to the fhir server. at line 441 in gen3_tracker/git/cli.py ```if os.path.isfile(bundle_file):``` the existing fhir bundle generated from the skeleton code is being used.


For what the ETL pod/job does I don't think it is appropriate to add bundle submission support to it especially since that is what the FHIR Server was developed for.